pokemon-showdown

```diff
@@ -170,14 +137,14 @@ if (!process.argv[2] || /^[0-9]+$/.test(process.argv[2])) {
 	case 'simulate-battle':
 		{
 			sucraseBuild();
-			var BattleTextStream = require('./.sim-dist/battle-stream').BattleTextStream;
-			var Streams = require('./.lib-dist/streams');
+			const BattleTextStream = require('./sim/battle-stream.js').BattleTextStream;
+			var Streams = require('./lib/streams');
 			var stdin = Streams.stdin();
-			var stdout = Streams.stdout();
+			const stdout = Streams.stdout();
```

Comment:
In the future, please do these kinds of changes in a separate commit, it makes it harder for me to review the change.

Reply:
Oh, I think this was just eslint --fix. I'll revert that in a sec.

Reply:
Reverted.